From 5546de157c407d6f31faffd8763754ee633d0e08 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Mon, 5 Oct 2015 14:33:35 +0200
Subject: [PATCH] Epic bugfix of doom MK2

---
 foofile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/foofile b/foofile
index fd11acc..93aa8ad 100644
--- a/foofile
+++ b/foofile
@@ -1 +1 @@
-#huehue, change
+#lol, another change
